docs/fedora-releases-guide/releasing-multiple-packages.md

```diff
@@ -0,0 +1,198 @@
+---
+title: Releasing multiple packages in a single update
+sidebar_position: 7
+---
+# Releasing multiple packages in a single update
```

Comment:
Very nicely written and explained! I like the 3 scenario structure and that we can have a link for each.